Resources/collectionApp_2020-02-05.sql

```diff
@@ -7,7 +7,7 @@
 #
 # Host: 127.0.0.1 (MySQL 5.7.29)
 # Database: collectionApp
```

Comment:
rename your db chris_wine_collection so its clear for Charlie and make sure to update this in your PDO connection too